docs/reference/speed_python.rst

```diff
@@ -91,9 +91,11 @@ code these should be pre-allocated and passed as arguments or as bound objects.
 
 When passing slices of objects such as `bytearray` instances, Python creates
 a copy which involves allocation of the size proportional to the size of slice.
-This can be alleviated using a `memoryview` object. `memoryview` itself
-is allocated on heap, but is a small, fixed-size object, regardless of the size
-of slice it points too.
+This can be alleviated using a `memoryview` object. The `memoryview` itself
+is allocated on the heap, but is a small, fixed-size object, regardless of the size
+of slice it points too. Slicing a `memoryview` creates a new `memoryview`, so this
+cannot be done in an interrupt service routine. Further, the slice syntax `a:b`
+causes further allocation by instantiating a `slice(a, b)` object.
```

Comment:
The single backticks are causing a build failure:

```
/home/runner/work/micropython/micropython/docs/reference/speed_python.rst:92: WARNING: 'any' reference target not found: a:b
/home/runner/work/micropython/micropython/docs/reference/speed_python.rst:92: WARNING: 'any' reference target not found: slice(a, b)
```

Unlike Markdown, Restructured Text needs double-backticks for inline code formatting.

Reply:
Thank you.